flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -776,6 +810,12 @@ private ApplicationReport startAppMaster(
 					Path.CUR_DIR);
 		}
 
+		if (!shipArchives.isEmpty()) {
+			fileUploader.registerMultipleLocalResources(
```

Comment:
Why not putting one argument per line? Normally we go for either all arguments on one line, or one per line. :)